cmd/gsm-secret-sync/main.go

```diff
@@ -56,13 +81,23 @@ func main() {
 	if err := o.Validate(); err != nil {
 		logrus.WithError(err).Fatal("Failed to validate options")
 	}
-	if err := o.setupLogger(); err != nil {
+
+	censor := secrets.NewDynamicCensor()
```

Comment:
In some point you should add the sensitive data into the censor via https://github.com/openshift/ci-tools/blob/main/pkg/secrets/censor.go#L30

Reply:
Adding them here https://github.com/openshift/ci-tools/pull/4667/files#diff-f76efbacdffe8d72345f140e5e9362365918a9932e8e3085cf20b2ad3d618771R90